data_structures/queue/queue.c

```diff
@@ -61,8 +61,10 @@ int deque()
         if (head->pre == NULL)
             head = NULL;
         else
+        {
             head = head->pre;
-        head->next = NULL;
+            head->next = NULL;
```

Comment:
There don't seem to be any changes rather than the brackets. 🤔
Could you please explain if you made any changes?

Reply:
hm.. I think there is no bug. The `next` element of the `pre` should be NULL in this case. (One thing left is to `free`, but it's  another).